lib/pact/consumer_contract/pact_file.rb

```diff
@@ -81,7 +81,11 @@ def get_remote(uri, options)
       request = Net::HTTP::Get.new(uri)
       request.basic_auth(options[:username], options[:password]) if options[:username]
       request['Authorization'] = "Bearer #{options[:token]}" if options[:token]
-      Net::HTTP.start(uri.host, uri.port, :ENV, use_ssl: uri.scheme == 'https') do |http|
+      http = Net::HTTP.new(uri.host, uri.port, :ENV)
+      http.use_ssl = (uri.scheme == 'https')
+      http.ca_file = ENV['SSL_CERT_FILE'] if ENV['SSL_CERT_FILE'] && ENV['SSL_CERT_FILE'] != ''
```

Comment:
I don't think these two lines are necessary if you've already set `:ENV` as the third parameter.

Reply:
Oh, no, it's the proxy env var that gets set via `:ENV`! My bad.